externals/coda-oss/modules/c++/io/include/io/SeekableStreams.h

```diff
@@ -1,7 +1,7 @@
 /* =========================================================================
```

Comment:
Can you do `git checkout --externals` to get rid of the rest of these changes?

Reply:
This still needs to be addressed

Reply:
I was looking this up because the command seemed to be not doing anything.
Is this essentially to checkout the files from the externals directory from the master branch so i don't get any differences?
If so should the command be `git checkout master -- externals` ?

Reply:
Oops, that command would have been to get rid of uncommitted changes in the externals directory. No good here.  Maybe you want `git checkout <commit_hash> <file>`?